yjit/src/codegen.rs

```diff
@@ -4133,6 +4153,87 @@ fn gen_return_branch(
     }
 }
 
+
+fn pop_args_splat(argc: i32, ctx: &mut Context, asm: &mut Assembler, ocb: &mut OutlinedCb, side_exit: CodePtr, num_params: u32) -> i32 {
```

Comment:
You should write a comment above this function to explain what it does a little bit.